packages/gateway-trpc/src/modules/job/job-router.ts

```diff
@@ -0,0 +1,45 @@
+import { JobSchema, JobWriteSchema } from "@dotkomonline/types"
+import { z } from "zod"
+import { adminProcedure, t } from "../../trpc"
+
+export const jobRouter = t.router({
```

Comment:
A job executor is usually called from another component in a system, we might not want/need a router for this